testing/regression/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "dependencies": {
+    "nconf": "^0.10.0",
+    "testcafe": "^0.20.5"
```

Comment:
Try to use the latest 1.2.0 version

Reply:
It will be better move testcafe dependency to main project.json in devDependencies